src/otelcollector/otelcol-config.yml

```diff
@@ -3,12 +3,16 @@ receivers:
     protocols:
       grpc:
       http:
+
 exporters:
   jaeger:
     endpoint: "jaeger:14250"
     tls:
       insecure: true
   logging:
+  prometheus:
+    endpoint: 0.0.0.0:9464
```

Comment:
```suggestion
    endpoint: "otelcol:9464"
```
nit: To use the same notation as above (if this works)

Reply:
Done, I tested, it works.

![image](https://user-images.githubusercontent.com/11855957/174920546-b1164023-b724-4d8d-a3e8-efaf676858a4.png)